### DISCUSSION TITLE
Getting specific player stats live

### DISCUSSION THREAD

**Discussion by k3ls3yP** [cwendt94/espn-api#375] [ESPN API] [answered]
Hello!
I'm very much enjoying this API!
I'm making a discord bot and my folks asked me to make a command where they can do the specific player's stats of the current week. I see that league.player_info  gives the season stats, but doesn't look like it gives live data (correct me if I"m wrong please!) Or at least for this current week (4) there's no stats (even empty) given in the array spot 4. (I verified that array 3 is week 3's data.) 
I do see that there's a class called Box Player that has the info I need, but is this only for the free_agents, and then when gotten from Box Score. Would I just need to combine these into a list and sort through them?

Basically I just need a way to put in a specific player name and get the live stats for them. Advice please? Thank you!

**Comment by dtcarls** [comment, marked as answer]
boxscore.home_lineup.player.points (also works for away_lineup)
Check out here https://github.com/dtcarls/fantasy_football_chat_bot/blob/2c9c7347010137fbcde2d967d2dfa4e275ec9290/ff_bot/ff_bot.py#L182
Not sure if there’s a way to pull the player only without the lineups.

**Reply by cwendt94** [reply]
Following up on @dtcarls answer, getting the box score `league.box_scores()` will give you all of the players stats for that week. You will just need to go through each matchup and look at teams lineup. 

You also can see the players stats for that week just from looking at your teams roster. So it would look something like this
`league.teams[0].roster[0].stats[week]`.

**Comment by dtcarls** [comment]
Potentially the Player.stats dict 
https://github.com/cwendt94/espn-api/wiki/Player-Class
